_layout/head.html

```diff
@@ -14,9 +14,14 @@
 
   {{isdef title}} <title>{{fill title}}</title>   {{end}}
 
-  {{ispage /blog/* /jsoc/gsoc/*}}
+  {{ispage /blog/* /jsoc/gsoc/* /contribute/* /downloads/* /community/*}}
   <style>
 	  .container ul li p {margin-bottom: 0;}
+		.container ol li p {margin-bottom: 0;}
+		.container ul ul {margin: .4em 0 .4em 0;}
+		.container ul ol {margin: .4em 0 .4em 0;}
+		.container ol ul {margin: .4em 0 .4em 0;}
+		.container ol ol {margin: .4em 0 .4em 0;}
```

Comment:
`ul li p` and `ol li p` is to combat Julia's Markdown module. The rest is to make 1-level nested lists have a bit of space to make them somewhat more readable